ingestion/src/metadata/profiler/metrics/static/min.py

```diff
@@ -98,7 +98,9 @@ def df_fn(self, dfs=None):
         if is_quantifiable(self.col.type):
             return min((df[self.col.name].min() for df in dfs))
         if is_date_time(self.col.type):
```

Comment:
should we add the type to `is_date_time`. I am assuming if the type is already a datetime type then we should not have to convert it to a pd.datetime at runtime right?

Reply:
min_ = min((pd.to_datetime(df[self.col.name]).min() for df in dfs))
      if pd.isna(min_):
          return None
      return int(min_.timestamp() * 1000)
      
as the data is not parsed as datetime always, instead it sometimes is parsed as string
`YYYY-MM-DD HH-MM-SS` pandas considers it datetime, whereas when we try to parse `YYYY-MM-DD` using pandas, it evals it as an object / string..
one of the reasons why we need to convert it to datetime